microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -18,7 +18,7 @@ For an overview, see [Using the REST API for updating Microsoft Edge Add-ons](./
 <!-- ------------------------------ -->
 #### Versions of the Update REST API
 
-As of September 6, 2024, both v1.1 and v1 of this Update REST API are supported.  Later, v1 will no longer be supported.  The date for ending v1 support is not yet finalized.
```

Comment:
We should add -

November 2024 Update - Support for v1 will end on December 31st 2024.

Reply:
in roadmap.md added-

## Deprecation of the Update REST API v1

As of September 6, 2024, both v1.1 and v1 of the Update REST API are supported.  Support for v1 will end on December 31, 2024.

See [Using the REST API for updating Microsoft Edge Add-ons](../publish/api/using-addons-api.md).